optaplanner-examples/src/main/java/org/optaplanner/examples/common/experimental/api/Sequence.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.examples.common.experimental.api;
+
+public interface Sequence<ValueType_, DifferenceType_ extends Comparable<DifferenceType_>> {
+    /**
+     * Returns the first item in the sequence
+     * 
+     * @return Never null, the first item in the sequence
```

Comment:
I hate this nitpicking, but this is not a sentence, so it should not start with a capital letter.

Reply:
Both these comments apply to the entire interface.